src-kmerge/KMerge/Heap.hs

```diff
@@ -41,7 +42,7 @@ placeholder :: a
 placeholder = unsafeCoerce ()
 
 -- | Create new heap, and immediately extract its minimum value.
-newMutableHeap :: forall a m. (PrimMonad m, Ord a) => [a] -> m (MutableHeap (PrimState m) a, Maybe a)
+newMutableHeap :: forall a m. (PrimMonad m, Ord a) => NonEmpty a -> m (MutableHeap (PrimState m) a, a)
```

Comment:
this "breaks" API compatibility with loser tree.

Either change both, or dont change this.

Reply:
Thanks for bringing this to my attention! I was unaware that the `LoserTree` module shared an API interface with the `Heap` module. I have unified the exposed cross-module API.